ldj-spring-renderer-example/src/main/java/com/infosupport/ldoc/springexample/util/PlantUmlBuilder.java

```diff
@@ -0,0 +1,16 @@
+package com.infosupport.ldoc.springexample.util;
+
+import java.io.PrintWriter;
+
+public class PlantUmlBuilder {
```

Comment:
Well I want to avoid at least exact method code duplications between the event and amqp sample renderers. 
As we talked about, if we merge the interaction model for amqp and events, the code for generating plantUML from the interaction model would be identical. So if we decide to do that, we will probably have all of that code in a common class.
So I would wait for that decision before addressing this issue